src/main/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/MolecularFunctionAnnotationBuilder.java

```diff
@@ -15,177 +16,186 @@
     private static final Logger logger = LogManager.getLogger();
 
```

Comment:
This class only has static methods - it's a utility class. Should probably have a private no-op constructor.

Reply:
I think many of the classes are composed of only static methods, but I would like to rework it in a future PR to make some methods instance methods.

Reply:
So an _instance_ of this builder would build a single Molecular Function Annotation?

Reply:
Well, an instance of this builder would build all molecular function annotations but as an object instead of through static methods.

Reply:
Hmm. Sounds like this is heading in the direction of a singleton. What advantage would there be to a single instance vs. static methods in a utilities class?

Reply:
Keeping the class and methods static might be a way to go.  Another PR for reorganization can look at this.